account_cost_center/security/ir.model.access.csv

```diff
@@ -0,0 +1,4 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_account_cost_center_invoice,account.cost.center,model_account_cost_center,account.group_account_invoice,1,1,1,1
+access_account_cost_center_user,account.cost.center,model_account_cost_center,account.group_account_user,1,1,1,1
```

Comment:
that's in line with having analytic accounts being fully accessible to account_user, so yes, makes sense